hca_bundle_tools/file_metadata_to_csv.py

```diff
@@ -24,19 +24,20 @@ def __init__(self, order = None, ignore = None, format_filter = None):
              "path",
              "^file_name",
              "^file_format",
-             "^sequence_file. *",
-             "^analysis_file. *",
-             "^donor_organism. *",
-             "^specimen_from_organism. *",
-             "^cell_suspension. *",
-             "^. * protocol. *",
+             "^sequence_file.*",
+             "^analysis_file.*",
+             "^donor_organism.*",
+             "^specimen_from_organism.*",
+             "^cell_suspension.*",
+             "^.*protocol.*",
              "^project."
         ]
 
         self.default_ignore = ignore if ignore else [
             "describedBy",
             "schema_type",
-            "provenance",
```

Comment:
@hannes-ucsc by default anything in the `provenance` block is ignored when we build the TSV (including the document id). I changed it so only the `submission_date` and `update_date` are ignore now.

Reply:
I usually keep unrelated changes in separate commits.

Reply:
not sure how those regexes got screwed up in the commit. Think my IDE did something!